### PR TITLE
feat: support admin version

### LINKED ISSUE #17: Add support for developers to set admin version
So extension users can make use of https://github.com/adobe/helix-sidekick/issues/240

### PR DESCRIPTION
**Pull request by rofe**
Fix #17

### CODE REVIEW

**Comment by rofe** [comment]
- Revamped options UI will collapsable areas
- Added new area for development features (incl. "test sidekick locally")
Screenshot 2022-01-29 at 15 49 05

File: src/extension/_locales/fr/messages.json [hunk 1]
```diff
@@ -164,7 +164,22 @@
...
+    "message": "Invoke a local development version of the Sidekick"
+  },
+  "admin_version": {
+    "message": "Use a specific Helix Admin API version"
+  },
+  "admin_version_description": {
+    "message": "Caution: This can have unexpected side effects!"
+  },
+  "admin_version_placeholder": {
+    "message": "Admin Version: x.y.z"
```
Comment (rofe) [inline src/extension/_locales/fr/messages.json:183]: @kptdobe before I have to turn to Google Translation again, could you provide the proper French translations? 😄

File: src/extension/_locales/fr/messages.json [hunk 2]
```diff
@@ -164,7 +164,22 @@
...
     "message": "Avancées"
   },
+  "config_development": {
+    "message": "Développement"
+  },
   "developer_mode": {
     "message": "Tester le Helix Sidekick localement"
+  },
+  "developer_mode_description": {
+    "message": "Invoke a local development version of the Sidekick"
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:174]:
```suggestion
    "message": "Invoquer une version locale du Helix Sidekick"
```

File: src/extension/_locales/fr/messages.json [hunk 3]
```diff
@@ -164,7 +164,22 @@
...
+  },
+  "admin_version": {
+    "message": "Use a specific Helix Admin API version"
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:177]:
```suggestion
    "message": "Utiliser une versions spécifique de l'API Helix Admin"
```

File: src/extension/_locales/fr/messages.json [hunk 4]
```diff
@@ -164,7 +164,22 @@
...
+  },
+  "admin_version_description": {
+    "message": "Caution: This can have unexpected side effects!"
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:180]:
```suggestion
    "message": "Attention: ceci peut avoir des effets indésirables!"
```

File: src/extension/_locales/fr/messages.json (hunk 1 above)
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:183]:
```suggestion
    "message": "Version de l'Admin: x.y.z"
```

**Review by kptdobe — APPROVED** [review]

**Pull request merged**